<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NASLoginDialog</class>
 <widget class="QDialog" name="NASLoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>132</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>479</width>
    <height>132</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>479</width>
    <height>132</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>用户名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditUserName">
         <property name="minimumSize">
          <size>
           <width>305</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>305</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>密码：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPassword">
         <property name="minimumSize">
          <size>
           <width>305</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>305</width>
           <height>22</height>
          </size>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>记住密码</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="buttonLogin">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>登录</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCancel">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>NASLoginDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
